coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/analyzer/repository/FileRepository.java

```diff
@@ -62,4 +62,14 @@
       int frequency,
       @NonNull List<String> includes,
       @NonNull List<String> excludes);
+
+  /**
+   * @param projectId The id of the project.
+   * @param path The filepath to search for.
+   * @return True if any file with the given path exists.
+   */
+  @Query(
+      "MATCH (p)-[:CONTAINS]->(f:FileEntity) WHERE ID(p) = {0} AND f.path = {1} WITH f LIMIT 1 RETURN COUNT(f) > 0 ")
+  @NonNull
+  Boolean fileExistsByPath(long projectId, @NonNull String path);
```

Comment:
There is a query called `fileInPathExists` in the `ModuleRepository` that does something very similar. Could you move that query to this repository and rename the new query to `fileWithPathExists`
Also, both queries can return just `boolean` and the `@NonNull` annotation can then be removed.

Reply:
Done